data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApiTypes.java

```diff
@@ -104,6 +106,15 @@ public class BeaconRestApiTypes {
       new ParameterMetadata<>(
           SLOT, CoreTypes.UINT64_TYPE.withDescription("`uint64` value representing slot"));
 
+  public static final ParameterMetadata<UInt64> START_PERIOD_PARAMETER =
+      new ParameterMetadata<>(
+          START_PERIOD,
+          CoreTypes.UINT64_TYPE.withDescription("`uint64` value representing start_period"));
+
+  public static final ParameterMetadata<UInt64> COUNT_PARAMETER =
+      new ParameterMetadata<>(
+          COUNT, CoreTypes.UINT64_TYPE.withDescription("`uint64` value representing count"));
+
```

Comment:
We should probably size constrain these for sanity... I'm not sure if the spec has anything in it about acceptable ranges....
In testing we can play with both being MAX_VALUE and watching it go boom...
I've got no firm ideas about where we constrain them, just a future idea for something we will need to look at...

Reply:
Agreed. According to the [reference](https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/full-node.md#create_light_client_update),

> Full nodes SHOULD provide the best derivable LightClientUpdate (according to is_better_update) for each sync committee period covering any epochs in range [max(ALTAIR_FORK_EPOCH, current_epoch - MIN_EPOCHS_FOR_BLOCK_REQUESTS), current_epoch] where current_epoch is defined by the current wall-clock time. Full nodes MAY also provide LightClientUpdate for other sync committee periods.

In other words, we could constrain it according to the range of valid epochs.